public/scss/_error-list.scss

```diff
@@ -0,0 +1,13 @@
+.error-list {
+    padding: 15px;
+    background: #f3e9e8;
+    border-left: 4px solid $color-red;
+
+    &__header {
+        margin-top: 10px;
+    }
```

Comment:
I've added some CSS variables for spacing here: https://github.com/cds-snc/cra-claim-tax-benefits/blob/master/public/scss/_variables.scss#L11

For `height`s and `width`s, it doesn't really matter, but I think for `padding`s and `margin`s, we should try and use them to be consistent.

Also, let's put in your new red background as a new color.